app/templates/admin/services/service_list.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'admin/base.html' %}
+{% set page_title='Services' %}
+
+{% block action_buttons %}
+  <a href="/admin/services/adding" type="button" class="btn btn-sm btn-outline-primary">Add New</a>
+{% endblock %}
+
+{% block content %}
+  <table class="table table-hover table-striped table-sm">
+    <thead>
+      <tr>
+        <th scope="col">Name</th>
+        <th scope="col">Slug</th>
+        <th scope="col">Description</th>
+        <th scope="col">Price</th>
```

Comment:
hmmm, not sure how we gonna display prices here, I think having it inside a column is not suitable here